whatwg.org/index

```diff
@@ -1,39 +1,93 @@
+                        
+                        
+                  <h2>IRC</h2><p class="irc">Chat with other members of the WHATWG community</p></a>
+             <a class="span4" href="https://checker.html5.org/">
+                <svg width="101" height="101" viewBox="0 0 101 101" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><title>noun_974826_cc</title><use xlink:href="#w" transform="translate(21.45 29.35)"/><use xlink:href="#b" transform="translate(.004 .005)"/><use xlink:href="#c" transform="translate(21.45 29.35)"/><defs><path id="w" d="M21.48 44.74L0 23.25l11.4-11.4 10.31 11.21L49.49 0l8.3 8.44-36.31 36.3z"/><path id="b" d="M85.356 14.645C78.363 7.652 69.454 2.89 59.756.96 50.055-.97 40 .023 30.865 3.807S13.92 14 8.426 22.22C2.933 30.445 0 40.11 0 50s2.932 19.556 8.427 27.78C13.92 86 21.73 92.41 30.867 96.193c9.135 3.784 19.19 4.775 28.888 2.845 9.7-1.93 18.608-6.692 25.6-13.685 4.664-4.632 8.365-10.14 10.89-16.207C98.77 63.08 100.07 56.572 100.07 50c0-6.572-1.3-13.08-3.825-19.148-2.525-6.067-6.226-11.576-10.89-16.207zm-5.53 65.18c-7.95 7.732-18.62 12.024-29.71 11.948-11.088-.077-21.7-4.516-29.542-12.356-7.84-7.84-12.28-18.454-12.356-29.542-.076-11.09 4.216-21.762 11.948-29.71 7.948-7.732 18.62-12.024 29.71-11.948 11.088.077 21.7 4.515 29.54 12.356 7.842 7.84 12.28 18.454 12.357 29.542.077 11.09-4.215 21.762-11.947 29.71z"/><path id="c" d="M21.71 23.06L11.4 11.85 0 23.25l21.48 21.49 36.3-36.3L49.49 0 21.71 23.06z"/></defs></svg>
+                <h2>HTML checker</h2>
```

Comment:
Similarly these are not h2s. They can be `<strong>`, I guess, like the existing site